docs/manual/docs/cmd/spo/orgassets/orgassets-list.md

```diff
@@ -0,0 +1,29 @@
+# spo orgassetslibrary list
```

Comment:
This document is missing in the `docs/mkdocs.yam` list of urls. Could you please add it?
You can perhaps add the command in the powershell comparison list.